### DISCUSSION TITLE
Help to get this running

### DISCUSSION THREAD

**Discussion by jensapagen** [bitmagnet-io/bitmagnet#314] [General]
Hello there.

Spent about 2000 hours trying to run this on my site “torrent.my” - but obviously I am doing something completely wrong.

I need someone to help me fix this. I will of course pay for your effort. You'll have full access to everything you need. Right now I'm on Hostinger VPS but theres possibility to migrate into DigitalOcean.
 

// baeckjens@gmail.com

**Comment by Technetium1** [comment]
To start with, DNS isn't correct https://www.ssllabs.com/ssltest/analyze.html?d=torrent.my&hideResults=on

**Comment by mgdigital** [comment]
Hi, if I've understood then you're trying to run a public instance. I've explained here why this is a bad idea and also why linking to public instances is banned in this repository: https://github.com/bitmagnet-io/bitmagnet/discussions/282

For general support you can come onto Discord and we'll try to help, but please bear in mind we can't help you to set up a public instance: https://discord.gg/6mFNszX8qM
